app/views/employments/EndEmploymentView.scala.html

```diff
@@ -15,41 +15,88 @@
  *@
 
 @import includes.forms._
-@import includes.{cancelLink, ptaHeader}
+@import includes.{cancelLink}
 @import java.time.LocalDate
-@import uk.gov.hmrc.play.views.html.helpers._
+
 @import uk.gov.hmrc.tai.viewModels.employments.EmploymentViewModel
 
 @import scala.concurrent.ExecutionContext
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukDateInput
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.hmrcfrontend.views.config.HmrcPageHeadingLegend
+@import uk.gov.hmrc.hmrcfrontend.views.config.HmrcSectionCaption
+
+@this(mainTemplate: MainTempleteNew, form: FormWithCSRF,  govukDateInput : GovukDateInput, govukButton: GovukButton, govukErrorSummary: GovukErrorSummary)
 
-@this(mainTemplate: MainTemplate, form: FormWithCSRF)
+@(dateForm: Form[LocalDate], viewmodel: EmploymentViewModel)(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser,  ec: ExecutionContext)
 
-@(dateForm: Form[LocalDate], viewmodel: EmploymentViewModel)(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer,  ec: ExecutionContext)
 
-@header = @{
-    ptaHeader(
```

Comment:
was the header meant to be removed?

Reply:
yes its not used in pages where we are using a form as the H1 heading is handled by the Form field legend - this is to prevent duplication of  the heading and was originally an issue reported from the accessibility team. Also as we're unlikely to have the print button on the form pages  - there is no need for the header anymore